regolith/helpers/a_presentationhelper.py

```diff
@@ -98,6 +100,9 @@ def subparser(subpi):
     subpi.add_argument("--no_cal",
                        help=f"Do not add the presentation to google calendar",
                        action="store_true")
+    subpi.add_argument("--gitlab", 
```

Comment:
this should probably be more general.  What if people store their talk files in github?  Something like `--create_talk_repo`? 

I think for me the best behavior is if I can set this value in my global config (user.json) so it does it by default and this button would turn it off, but others may want it off by default.   Not sure how to do that, so maybe think about it.